mobile/examples/object_detection/android/app/src/main/java/ai/onnxruntime/example/objectdetection/MainActivity.kt

```diff
@@ -101,7 +101,7 @@ class MainActivity : AppCompatActivity() {
     private fun readClasses(): List<String> {
         val classes_id = R.raw.classes
         val classes_bytes = resources.openRawResource(classes_id).readBytes()
-        return String(classes_bytes).split("\r\n".toRegex())
+        return String(classes_bytes).split("\r?\n".toRegex())
```

Comment:
Can we use a https://docs.oracle.com/javase/8/docs/api/java/io/BufferedReader.html here instead? 

Or like this:

https://stackoverflow.com/questions/37308957/how-do-you-read-a-text-file-from-a-resources-folder-in-java-1-7

Reply:
something like this should work:
https://github.com/microsoft/onnxruntime-inference-examples/blob/6c6304169a75420b83547e3fedcfa706ea8b2ec4/mobile/examples/image_classification/android/app/src/main/java/ai/onnxruntime/example/imageclassifier/MainActivity.kt#L150

Reply:
@snnn @edgchen1 updated.